src/app/pages/HomePage/index.tsx

```diff
@@ -1,18 +1,86 @@
-import { FC } from 'react'
-import { Link } from 'react-router-dom'
-import Typography from '@mui/material/Typography'
+import { FC, useCallback, useState } from 'react'
+import { Logotype } from '../../components/PageLayout/Logotype'
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import background from './images/background.svg'
 import { COLORS } from '../../../styles/theme/colors'
+import { Search } from '../../components/Search'
+import { ParaTimeSelector } from './ParaTimeSelector'
+import { Footer } from '../../components/PageLayout/Footer'
+import { useNavigate } from 'react-router-dom'
+import { SearchUtils } from '../../components/Search/search-utils'
 
-export const HomePage: FC = () => (
-  <div>
-    <Typography variant="h1" color={COLORS.white}>
-      home
-    </Typography>
-    <div>
-      <Link to="/emerald/blocks">blocks</Link>
-    </div>
-    <div>
-      <Link to="/emerald">dashboard</Link>
-    </div>
-  </div>
-)
+const HomepageLayout = styled(Box)(() => ({
+  position: 'relative',
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100vw',
+  height: '100vh',
+  backgroundColor: COLORS.brandDark,
+  '&::before': {
+    content: '" "',
+    position: 'absolute',
+    inset: 0,
+    backgroundImage: `url("${background}")`,
+    backgroundSize: 'cover',
+    backgroundPosition: 'center',
+    backgroundRepeat: 'no-repeat',
+  },
```

Comment:
How come it's not just
```suggestion
  backgroundImage: `url("${background}")`,
  backgroundSize: 'cover',
  backgroundPosition: 'center',
  backgroundRepeat: 'no-repeat',
```

Reply:
Ah, neither works well over the fold
![image](https://user-images.githubusercontent.com/3758846/211093406-838ac7b4-c6ff-44e5-9df9-469af4be9a99.png)

Reply:
The reason for having background as a element on top, is because so that opacity of the background is correctly displayed. Did add min-height for now. But will deal with responsiveness in mobile PR.